vars/rollingUpgradePipeline.groovy

```diff
@@ -76,7 +76,10 @@ def call(Map pipelineParams) {
 
             // NOTE: Optional parameters for BYO ScyllaDB stage
             string(defaultValue: '',
-                   description: 'Custom "scylladb" repo to use. Leave empty if byo is not needed. If specified then need to define "base_versions" param explicitly.',
+                   description: (
+                       'Custom "scylladb" repo to use. Leave empty if byo is not needed. ' +
+                       'If set then it must be proper GH repo. Example: git@github.com:personal-username/scylla.git\n'
+                       'and, in case of an "rolling upgrade", need to define "base_versions" param explicitly.'),
```

Comment:
That's true regardless of this option, or I miss why you mention it here....

Reply:
`base_versions` is needed only for upgrade scenario.